HabitTracker.BatataDoc3/HabitTracker/HabitTrackerApp.cs

```diff
@@ -0,0 +1,444 @@
+﻿using HabitTracker.BatataDoc3.db;
+
+namespace HabitTracker.BatataDoc3.HabitTracker
```

Comment:
This class would love the extra space to the left. Use file-scoped namespace to give more room.